src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,38 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+  }
+
+  &__link {
+    position: relative;
+    color: $background-color;
+    text-decoration: none;
+    font-size: 22px;
+    line-height: 22px;
+    letter-spacing: 2px;
+
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: -8px;
+      display: block;
+      height: 1px;
+      width: 100%;
+      background-color: $background-color;
+
+      transition: transform 0.3s;
```

Comment:
you use this value often, you can create variable for 0.3s;
don't forget to replace it in othe places too